app/models/iiif_canvas_indexer_enqueuer.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+##
+# For a given object in an exhibit, gets the correct manifest, and enqueues
+# canvas/page indexing jobs from that manifest.
+class IiifCanvasIndexerEnqueuer
```

Comment:
I feel like this class is where the overlap with how the Traject+ Manifest > Canvas > AnnoList > Anno capture, mapping, and indexing ('transform') could be reused. I don't think how this PR does the work should change for that reason, but it would be a nice way to take some of this IIIF resource class logic and abstract out in the same Traject+ code for other semantic logic (like MODS, Bibtex, other). Also could help solve some of the iteration issues, if we can call Traject at a Anno-level (for example) within this, but again, tomato / tomahtoe.

Reply:
I've added a IIIF support at various levels issue to the Traject+ repo, fwiw.